src/pages/Profile.js

```diff
@@ -476,18 +469,6 @@ const Profile = ({ queryParams }) => {
 
             sectionTitle('Program Info'),
```

Comment:
With the non-profit status and PI/program lead gone, would it make sense to go ahead and change this section's label to "Location"?